Cubical/Data/Sum/Base.agda

```diff
@@ -9,8 +9,16 @@ data _⊎_ {ℓ ℓ'} (P : Set ℓ) (Q : Set ℓ') : Set (ℓ-max ℓ ℓ') wher
 
 private
   variable
-    ℓ : Level
-    A B C D : Set ℓ
+    ℓ ℓ' ℓ'' ℓ''' : Level
+    A : Set ℓ
+    B : Set ℓ'
+    C : Set ℓ''
+    D : Set ℓ'''
+
```

Comment:
You don't need all the ```ℓ' ℓ'' ℓ'''``` just using the single ```ℓ``` in the signatures of ```A```, ```B```, ```C```, and ```D``` would obtain the same result.

Reply:
Thanks. I have them removed. I should be more careful when using new features ...